tests/ssl-opt.sh

```diff
@@ -6927,6 +6966,7 @@ run_test    "Large client packet TLS 1.2 AEAD shorter tag" \
             -s "Read from client: $MAX_CONTENT_LEN bytes read"
 
 # Test for large server packets
+# The tests below fail when the server's OUT_CONTENT_LEN is less than 16384.
```

Comment:
So why is there no corresponding `requires`?

Reply:
I believe they would be out of scope for this PR. My reasoning is based on [this discussion](https://github.com/ARMmbed/mbedtls/pull/4695#discussion_r662995615), where we concluded that if it's a pre-existing issue (which this is), then it should be in a different PR. 

Before this PR, setting OUT_CONTENT_LEN to e.g. 8096 would not trigger the hard abort but still fail these tests.

The note was added for future reference. Feel free to let me know if this should be included in the scope as the line is definitely a little blurry.